account_banking_sepa_credit_transfer/migrations/16.0.1.2.0/post-migration.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2023 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    sct_method = env.ref("account_banking_sepa_credit_transfer.sepa_credit_transfer")
+    sct_method.write({"payment_order_ok": True})
```

Comment:
Don't force this. Just rename the column of the field on the source one.

Reply:
But the payment method for SEPA credit transfer should always have payment_order_ok = true, no ? Otherwise, what's the point of installing account_banking_sepa_credit_transfer ??